libs/components/lookup/src/lib/modules/lookup/fixtures/lookup-input-box.component.fixture.html

```diff
@@ -9,3 +9,4 @@
     ></sky-lookup>
   </sky-input-box>
 </form>
+<button class="sky-btn sky-btn-primary" id="test-button">Test</button>
```

Comment:
These are just for ease of testing things that used to use an event target but now use the document's active element.